### PR TITLE
Hydra-core supports Python 3.9

### PR DESCRIPTION
**Pull request by alanhdu**
See https://github.com/facebookresearch/hydra/blob/d097e2f089122d2c9c95cbc0fbfb63387aef2309/setup.py#L52

### CODE REVIEW

File: recipe/meta.yaml
Comment (omry) [inline recipe/meta.yaml]: can you update the deps to match the [requirements](https://github.com/facebookresearch/hydra/blob/1.0_branch/requirements/requirements.txt) of Hydra 1.0.5?

```
omegaconf>=2.0.5,<2.1
antlr4-python3-runtime==4.8
importlib-resources;python_version<'3.9'
```

**Review by omry — CHANGES_REQUESTED** [review]

**Comment by alanhdu** [comment]
I updated the omegaconf dependency, but am not sure how to update the `importlib-resources` dependency: https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python says that `noarch` doesn't support selectors, so I'm not sure how to represent that.

**Comment by omry** [comment]
Okay, let's see if anyone else has an idea.
importlib-resoruces is a weird one but I think it's pretty common.
there are probably other projects with the same issue.

**Comment by alanhdu** [comment]
Is there anything else I can do to help along the review? I looked into the `noarch` dependency on Python version selectors, but couldn't find any guidance (I would've assumed this was a common problem, but looking ta other `noarch` dependencies they seem to just install the extra dependency).

File: recipe/meta.yaml
Comment (omry) [inline recipe/meta.yaml:29]: Can you add a comment this is only needed in Python < 3.9?
At some point we will be able to remove it.

**Review by omry — APPROVED** [review]
If the standard approach is to just install the extra dependency I think we can go with it.
Please add a comment before merging.

**Review by omry — APPROVED** [review]
lgtm but I don't have merge access.
@hadim ?

**Comment by hadim** [comment]
@omry do you want to be a maintainer?

**Comment by omry** [comment]
I don't mind rubber stamping but I don't have the time to learn how to actually maintain the conda package.
feel free to add me but I would still need help.

**Comment by hadim** [comment]
Not much more what you already did in that PR actually :-)

**Pull request merged**
